docs/docs/configuration/connecting-to-data-stores/signalfx.md

```diff
@@ -25,7 +25,9 @@ You need your SignalFx:
 
 Follow this [guide](https://docs.splunk.com/Observability/references/organizations.html).
 
-![](https://res.cloudinary.com/djwdcmwdz/image/upload/v1674644337/Blogposts/Docs/screely-1674644332529_cks0lw.png)
+![SignalFX](../img/SignalFX-settings.png)
```

Comment:
When running the `npm run build` command in the `docs` folder I get an error about this image being missing, do you mind taking a look?

Reply:
Can you use it from this file upload?
<img width="1419" alt="SignalFX-settings" src="https://user-images.githubusercontent.com/97040417/231831245-5285fee3-b7f6-4dd2-a96d-9ce8e8e22bbc.png">